packages/addons/addon-depends/chrome-depends/gtk3/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="gtk3"
-PKG_VERSION="3.24.4"
-PKG_SHA256="d84f59ff02a87cc90c9df4a572a13eca4e3506e2bf511e2b9cbdb4526fa0cb9c"
+PKG_VERSION="3.94.0"
+PKG_SHA256="a947caa5296610b0f1d7a03b58df34765c227c577c78e683e75eea3251a67035"
```

Comment:
`3.94.0` is a random dev version from 2018, current is 3.99.4 - also a dev version, your gtk3 commit is not even unpacking :)
latest "stable" is `3.24.23`
here updating to latest stable + change to meson (to drop the patch)

```
# SPDX-License-Identifier: GPL-2.0-or-later
# Copyright (C) 2017 Escalade
# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)

PKG_NAME="gtk3"
PKG_VERSION="3.24.23"
PKG_SHA256="5d864d248357a2251545b3387b35942de5f66e4c66013f0962eb5cb6f8dae2b1"
PKG_LICENSE="LGPL"
PKG_SITE="http://www.gtk.org/"
PKG_URL="https://ftp.gnome.org/pub/gnome/sources/gtk+/${PKG_VERSION:0:4}/gtk+-$PKG_VERSION.tar.xz"
PKG_DEPENDS_TARGET="toolchain at-spi2-atk atk cairo gdk-pixbuf glib libX11 libXi libXrandr libepoxy pango"
PKG_DEPENDS_CONFIG="libXft pango gdk-pixbuf shared-mime-info"
PKG_LONGDESC="A library for creating graphical user interfaces for the X Window System."
PKG_TOOLCHAIN="meson"
PKG_BUILD_FLAGS="-sysroot"

PKG_MESON_OPTS_TARGET="-Dbroadway_backend=false \
                       -Dcloudproviders=false \
                       -Dcolord=no \
                       -Ddemos=false \
                       -Dgtk_doc=false \
                       -Dintrospection=false \
                       -Dman=false \
                       -Dprint_backends=auto \
                       -Dquartz_backend=false \
                       -Dtests=false \
                       -Dwayland_backend=false \
                       -Dwin32_backend=false \
                       -Dxinerama=no \
                       -Dbuiltin_immodules=yes \
                       -Dx11_backend=true"

pre_configure_target() {
  # $TOOLCHAIN/bin/glib-compile-resources requires $TOOLCHAIN/lib/libffi.so.6
  export LD_LIBRARY_PATH="$TOOLCHAIN/lib:$LD_LIBRARY_PATH"
  export GLIB_COMPILE_RESOURCES=glib-compile-resources GLIB_MKENUMS=glib-mkenums GLIB_GENMARSHAL=glib-genmarshal
}
```

Reply:
linux commit dropped (kernel)
dropping gtk3 raising as a separate PR
-- https://github.com/LibreELEC/LibreELEC.tv/pull/4694
dropping glibc/busybox as a separate PR
-- https://github.com/LibreELEC/LibreELEC.tv/pull/4693
dropping lame raising as a separate PR
-- https://github.com/LibreELEC/LibreELEC.tv/pull/4692